docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+<response-info timems="0.108" idlems="0.040" threads="3" lastid="00000000018D3800" lastname="LargeThreadPool-thread-33" />
+
+</exclusive-start>
+
+<concurrent-global-final id="12379" timestamp="2020-10-18T13:35:44.594" intervalms="516905.029" >
+
+<concurrent-trace-info reason="card cleaning threshold reached" tracedByMutators="200087048" tracedByHelpers="12164180" cardsCleaned="4966" workStackOverflowCount="0" />
+
+</concurrent-global-final>
+```
+
+The final increment of a global cycle is triggered when the card-cleaning threshold is reached, indicated by the `reason` attribute of the `<concurrent-trace-info>` element.
+
+In the next section that begins with the `gc-start` element, you can find information about the amount of memory available (`<mem-info>`) and where it is located in the heap. This snapshot is taken before the final increment's operations and sub-operations are run and can be compared with a similar snapshot that is taken afterwards to understand the effect on the heap. The child element attribute values of the`<mem>` and `<mem-info>` elements indicate the status of the memory. 
+
+**Note:** You can double check that the increment is associated with the GC global cycle in the example by checking the `contextid` attribute value matches the `id=12364` attribute value of the cycle's <gc-cycle> element.
+
+```xml
+<gc-start id="12380" type="global" contextid="12364" timestamp="2020-10-18T13:35:44.594">
+
+<mem-info id="12381" free="277048640" total="1073741824" percent="25">
+
+<mem type="nursery" free="234609440" total="268435456" percent="87">
+
+<mem type="allocate" free="234609440" total="241565696" percent="97" />
+
+<mem type="survivor" free="0" total="26869760" percent="0" />
+
+</mem>
+
+<mem type="tenure" free="42439200" total="805306368" percent="5">
+
+<mem type="soa" free="2173472" total="765040640" percent="0" />
+
+<mem type="loa" free="40265728" total="40265728" percent="100" />
+
+</mem>
+
+<pending-finalizers system="0" default="0" reference="405" classloader="0" />
+
+<remembered-set count="17388" />
+
+</mem-info>
+
+</gc-start>
+
+<allocation-stats totalBytes="827488" >
+
+<allocated-bytes non-tlh="96" tlh="827392" />
+
+<largest-consumer threadName="LargeThreadPool-thread-68" threadId="00000000013D6900" bytes="65632" />
+
+</allocation-stats>
+```
+
+For this example, at the start of this GC increment, 25% of the total heap is available as free memory. This free memory is split between the following areas of the heap:
+
+- The *nursery* area, which has 87% of its total memory available as free memory. The free memory is only available in the *allocate* space of the *nursery* area. The *survivor* space has no free memory.
+
+- The *tenure* area, which has 5% of its total memory available as free memory. All of this free memory is in the long object allocation area(LOA). No free memory is available in the short object allocation area(SOA).
```

Comment:
we actually call them _large_ and _small_ object areas.

Reply:
Actually there is a bit of free memory in SOA (but rounded down to 0%). You could use _most_ instead of _all_ (for LOA) and _almost no_ instead of _no_

Reply:
You can also use work 'expectedly' somewhere (it's expected that tenure/SOA is low on free memory at this point).